### ISSUE TITLE
Access denied when trying to access the docs

### ISSUE THREAD

**Issue by emeraldtip** [SpartanJ/eepp#18]
This is what appears when I try to access the docs
![image](https://user-images.githubusercontent.com/48155462/229454892-6d0cb46d-59b0-4240-8619-41777fe968a3.png)

**Comment by SpartanJ** [comment]
Good catch, thanks for letting me know. It should point to index.html, otherwise won't work.
